src/app/cases/page.tsx

```diff
@@ -44,73 +54,90 @@ export default function Page() {
     });
   }, []);
 
+  const resetFilters = () => {
+    setCaseFilters({
+      remote: new Set(),
+      role: new Set(),
+      agency: new Set(),
+      languages: new Set(),
+      countries: new Set(),
+    });
+  };
+
   return (
     <PageContainer>
-      <div>
-        <H2>Browse Available Cases</H2>
-        <FiltersContainer>
-          <FilterDropdown
-            defaultValue="Remote/In Person"
-            options={['Remote/In Person', 'Remote Only', 'In Person Only']}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, remote: v as string })
-            }
-          />
-          <FilterDropdown
-            defaultValue="Attorney/Interpreter"
-            options={['Attorney/Interpreter', 'Interpreter Only']}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, role: v as string })
-            }
-          />
-          <FilterDropdown
-            defaultValue="Anywhere"
-            // requires knowing user location
-            options={[
-              'Within 5 miles',
-              'Within 10 miles',
-              'Within 20 miles',
-              'Within 100 miles',
-              'Anywhere',
-            ]}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, distance: v as string })
-            }
-          />
-          <FilterDropdown
-            defaultValue="All countries"
-            multi
-            options={[
-              'All countries',
-              ...Array.from(
-                new Set(
-                  caseData
-                    .filter(c => c.country)
-                    .map(c => (c.country ? c.country : '')),
-                ),
-              ),
-            ]}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, countries: v as string[] })
-            }
-          />
-        </FiltersContainer>
-      </div>
+      <H2>Browse Available Cases</H2>
+      <FiltersContainer>
+        <FilterDropdown
+          placeholder={defaultFilterValues.remote}
+          multi
+          options={new Set(['Remote', 'In Person'])}
+          value={caseFilters.remote}
+          fullText="All Remote/In Person"
```

Comment:
ok i know i said we should do this....but now that i actually test it, the wording feels a bit strange, i can't really tell if "all" applies to the options remote & in-person or to the whole set of cases displayed

@davidqing6432 thoughts on making the fullText be `Remote, In Person`? or something else? jdifjeiwfjew[f idk